__tests__/app.test.js

```diff
@@ -354,5 +354,35 @@ describe("app", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+    describe("GET - /api/articles?topic= filtering by query", () => {
+      test("GET: 200 - should filter the articles by the topic 'mitch'", () => {
+        return request(app)
+          .get("/api/articles?topic=mitch")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body.articles;
+            articles.forEach((article) => {
```

Comment:
I would add a test for the length of the array just before the forEach. If the array your receive from the response is empty then the tests inside your forEach would be skipped but the length test would still run